fanficfare/story.py

```diff
@@ -194,9 +194,17 @@ def no_convert_image(url,data):
         # parameter.
         ext = url[url.rfind('.')+1:].lower()
         if ext not in imagetypes:
-            logger.info("no_convert_image url:%s - no known extension -- using .jpg"%url)
-            # doesn't have extension? use jpg.
-            ext='jpg'
+            try:
+                from PIL import Image
```

Comment:
FFF doesn't require pillow(for PIL) for CLI users who don't want images.

Are you happy with the existing error reporting for not having PIL?
```
FFF: INFO: 2024-05-11 13:35:07,828: story.py(1670): Failed to load or convert image,
parent:http://test1.com?sid=1010&chapter=1
skipping:file:///C:/Users/uname/fff logo images/film-reel.jpeq
Exception: No module named 'PIL'
```